.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,153 @@
+                    || touch ${{ env.test_results_directory }}/fail.txt && exit 0
+              working-directory:  ${{ env.browser }}
+            - name: Prettyprint test report
+              id: test-report
+              run: |
+                xchtmlreport ${{ env.test_results_directory }}/results.xcresult
+            - name: Authenticate to Google Cloud
+              id: auth-gcloud
+              uses: google-github-actions/auth@v2.1.3
+              with:
+                credentials_json: ${{ secrets.GCP_SA_KEY }}
+            - name: Upload report to Google Cloud bucket
+              id: upload-gcloud
+              uses: google-github-actions/upload-cloud-storage@v2.1.0
+              with:
+               process_gcloudignore: false
+               path: '${{ env.test_results_directory }}/index.html'
+               destination: 'mobile-reports/public/${{ env.browser }}/${{ env.xcodebuild_test_plan }}/result_${{ github.run_number }}'
+            - name: Upload xcresult file to Github
+              id: upload-github
+              uses: actions/upload-artifact@v4.3.3
+              with:
+                name: ${{ env.browser }}-${{ env.xcodebuild_test_plan }}-${{ github.run_number }}
+                path: ${{ env.test_results_directory }}/index.html
+            - name : Process pass/fail info
+              id: pass-fail
+              run: |
+                # fail.txt is created when at least one test fails.
+                # job.status can't be used because the subsequent steps are not
+                # executed once a step fails.
+                if [ -e ${{ env.test_results_directory }}/fail.txt ]; then
+                    echo "pass_fail=❌" >> "$GITHUB_OUTPUT"
+                    echo "exit_value=1" >> "$GITHUB_OUTPUT"
+                else
+                    echo "pass_fail=✅" >> "$GITHUB_OUTPUT"
+                    echo "exit_value=0" >> "$GITHUB_OUTPUT"
+                fi
+            - name: Report to Slack
+              id: slack
+              uses: slackapi/slack-github-action@v1.26.0
+              with:
+                payload: |
+                  {
+                    "blocks":[
+                      {
+                        "type": "header",
+                        "text": {
+                          "type": "plain_text",
+                          "text": "${{ steps.pass-fail.outputs.pass_fail }} ${{ env.browser }} :firefox: ${{ env.xcodebuild_test_plan }} - ${{ env.ios_simulator }} iOS ${{ env.ios_version }}",
+                          "emoji": true
+                        }
+                      },
+                      {
+                        "type":"divider"
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                          "type": "mrkdwn",
+                          "text": "*Branch*: `${{ github.ref_name }}`"
```

Comment:
I would add all these fields (Test Plan, Branch, Simulator, Test Report... ) in a more compact way so that the notification looks more readable. From my point of view, everything looks really big in it.. 
Please look at how it was done in this [notification](https://github.com/mozilla-mobile/firefox-ios/blob/6a3bf6db3949c19ed14946765dfd955933b412cf/test-fixtures/ci/slack-notification-payload-autofill.json#L21) in case that helps. 
In that one the footer is also shown smaller, which makes it nicer and focus on the important details above.

Reply:
Would a Slack message like this look better to you? I can also remove the duplicated info from the header.
<img width="460" alt="Screenshot 2024-06-19 at 11 30 46 AM" src="https://github.com/mozilla-mobile/firefox-ios/assets/1740517/40ba95fb-43e9-4411-8f20-63002dd978a7">